src/com/facebook/buck/util/concurrent/BUCK.autodeps

```diff
@@ -9,6 +9,7 @@
       "//src/com/facebook/buck/util/immutables:immutables",
       "//third-party/java/concurrent-locks:concurrent-locks",
       "//third-party/java/guava:guava",
+      "//third-party/java/errorprone:error-prone-annotations",
```

Comment:
It was also unclear how to convince `buck autodeps` to add this dependency here? So i added it manually for now, and only here, ... Running `buck autodeps` would overwrite this file and remove it again, though.